### PR TITLE
PXBLUE-2655 Rebranded symbols mui demo

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes # PXBLUE-2655

#### Changes proposed in this Pull Request:
- pxblue ==> brightlayer-ui

### CODE REVIEW

**Comment by emclaug2** [comment]
@bkarambe I'm seeing this whenever I started the demo.  @jeffvg Do you have any issues running this?

```
Could not find a declaration file for module '@brightlayer-ui/symbols-mui'. 'C:/Users/Evan/Documents/git/pxblue/symbols/symbols-mui/demos/s
howcase/node_modules/@brightlayer-ui/symbols-mui/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/brightlayer-ui__symbols-mui` if it exists or add a new declaration (.d.ts) file containing `declare module '
@brightlayer-ui/symbols-mui';`  TS7016

    1 | import React from 'react';
  > 2 | import * as Symbols from '@brightlayer-ui/symbols-mui';
      |                          ^
    3 |
    4 | export const App = (): JSX.Element => (
    5 |     <div data-testid="symbolContainer">

```

**Comment by bkarambe** [comment]
> 

I tried running demo after this comment, and I am not getting any errors. Showcase demo is running successfully.

**Comment by emclaug2** [comment]
> > 
> 
> I tried running demo after this comment, and I am not getting any errors. Showcase demo is running successfully.

What are the commands you've used to run the demo?

**Comment by ektasawant** [comment]
> > > 
> > 
> > 
> > I tried running demo after this comment, and I am not getting any errors. Showcase demo is running successfully.
> 
> What are the commands you've used to run the demo?

Working fine for me. Tried `yarn start` from `symbols-mui/demos/showcase`.
Screenshot 2021-11-23 at 5 10 47 PM

File: symbols/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (ektasawant) [inline symbols/yarn.lock:2]: Can you please check this file needs to commit or not?

Comment (bkarambe) [reply symbols/yarn.lock:2]: Sure. @emclaug2 @daileytj

Comment (huayunh) [reply symbols/yarn.lock:2]: It looks like you run `yarn` by accident. This repository has a `package.json` file, but its dependencies list is empty; a `yarn.lock` file is not needed in the `symbols` folder.

**Pull request merged**